make/jdk/src/classes/build/tools/generatelsrequivmaps/EquivMapsGenerator.java

```diff
@@ -136,10 +136,29 @@ private static void processDeprecatedData(String type,
             }
         } else { // language, extlang, legacy, and redundant
             if (!initialLanguageMap.containsKey(preferred)) {
-                sb = new StringBuilder(preferred);
-                sb.append(',');
-                sb.append(tag);
-                initialLanguageMap.put(preferred, sb);
+                // IANA update 4/13 introduced case where a preferred value
+                // can have a preferred value itself.
+                // eg: ar-ajp has pref ajp which has pref apc
+                boolean foundInOther = false;
+                Pattern pattern = Pattern.compile("\\b"+preferred+"\\b");
```

Comment:
I think we should explicitly find a pattern with `,` instead of the word boundary. Otherwise, it could match `-` in a tag.

Reply:
Good point, changed it to match starting with `,` and ending with `,` or end of string boundary